ProjectMars/Pages/LoginPage.cs

```diff
@@ -0,0 +1,36 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using ProjectMars.Utilities;
+using System;
+using System.Collections.Generic;
+using System.Drawing.Printing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ProjectMars.Pages
+{
+    public class LoginPage
+    {
+        private IWebDriver webDriver;
+        public LoginPage(IWebDriver driver)
```

Comment:
this is not wrong what you have done here but you could extend to common driver for future reference, this way you do not have to pass the driver around to the constructor